server/views/pages/contacts/manage/address/addressType.njk

```diff
@@ -0,0 +1,51 @@
+{% extends "../../../../partials/layout.njk" %}
+{% from "../../../../partials/miniProfile/macro.njk" import miniProfile %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/radios/macro.njk" import govukRadios %}
+
+{% set title = "What type of address do you want to add for " + (journey.contactNames | formatNameFirstNameFirst) + "?" %}
+{% set pageTitle = applicationName + title %}
+{% set mainClasses = "app-container govuk-body" %}
+
+{% block content %}
+
+    {% include '../../../../partials/navigation.njk' %}
+    {{ miniProfile(prisonerDetails) }}
+    {% include '../../../../partials/formErrorSummary.njk' %}
+    <span class="govuk-caption-l">Contacts</span>
+    <h1 class="govuk-heading-l" data-qa="main-heading">{{ title }}</h1>
+    <div class="govuk-grid-row">
+        <div class="govuk-grid-column-three-quarters">
+            <form method='POST'>
+                <input type="hidden" name="_csrf" value="{{ csrfToken }}" />
+                {% set defaultOptions = [
+                    { divider: 'or' },
+                    {
+                        value: "DO_NOT_KNOW",
+                        text: "I don't know",
+                        checked: journey.addressType === 'DO_NOT_KNOW'
+                    }
```

Comment:
Ah - there is an I do not know option..

Reply:
Yeah, that's just to make them choose a real option or DO_NOT_KNOW. It'll translate to null when we call the API